cmd/snap/cmd_snap_op.go

```diff
@@ -122,6 +122,8 @@ var (
 	shortRemoveHelp  = i18n.G("Remove a snap from the system")
 	shortRefreshHelp = i18n.G("Refresh a snap in the system")
 	shortTryHelp     = i18n.G("Try an unpacked snap in the system")
+	shortEnableHelp  = i18n.G("Enable a snap in the system")
+	shortDisableHelp = i18n.G("Disable a snap in the system")
```

Comment:
Third person on all of them please (Removes, Refreshes, Tests (rather than tries, which is a bit strange)). Let's see if we can standardize on that.